README.md

```diff
@@ -1,33 +1,38 @@
 ![CI](https://github.com/reaganlo/nn-hal/actions/workflows/ci.yml/badge.svg)
 
-# Android Neural Networks HAL with OpenVINO supporting hardware accelerators such as /
+# ChromeOS Neural Networks HAL with OpenVINO supporting hardware accelerators such as /
 Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN)
+Intel® Movidius™ Vision Processing Units (VPUs)
 
 ## Introduction
-The Android Neural Network Hardware Abstraction Layer(NN HAL) provides the hardware accelration
-for Android Neural Networks (NN) API. Intel NN-HAL takes the advantage of the Intel MKLD-DNN,
-enables high performance and low power implementation of Neural Networks API.
-Intel MKL-DNN https://github.com/intel/mkl-dnn &  https://01.org/mkl-dnn
-Android NN API is on [Neural Networks API]
-(https://developer.android.com/ndk/guides/neuralnetworks/index.html).
-OpenVINO deep learning framework https://github.com/opencv/dldt & https://01.org/openvinotoolkit
-
+The ChromeOS Neural Network Hardware Abstraction Layer(NN HAL) provides the hardware accelration
+for ChromeOS Neural Networks (NN) API. Intel NN-HAL takes the advantage of the Intel OpenVINO CPUPlugin,
+based on MKLDNN which enables high performance and low power implementation of Neural Networks API.
+Intel OpenVINO is available at https://github.com/openvinotoolkit/openvino
```

Comment:
@anikulk Can we add a line specifying the exact version of OV (commit) that this version of the package depends on?